contribution-guide.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: ballerina-inner-page
+title: The Ballerina Contribution Guide
+permalink: /contribution-guide/
+---
+
+# The Ballerina Contribution Guide
+
+Ballerina is an open source programming language and platform for cloud-era application programmers to easily write software. It is an open-source project under the Apache license and the work of hundreds of contributors.
+
+We appreciate your help!
+
+- [Getting engaged](#getting-engaged)
+- [Getting started](#getting-started)
+- [Obtaining the source code](#obtaining-the-source-code)
+- [Building the project](building-the-project)
+- [Setting up the developer environment](setting-up-the-developer-environment)
+    - [Setting up IntelliJ IDEA](#setting-up-intelliJ-idea)
+    - [Setting up Eclipse](#setting-up-eclipse)
+- [Contributing to Ballerina grammar](#contributing-to-ballerina-grammar)
+- [Submitting your contribution](#submitting-your-contribution)
+- [Accepting the Contributor License Agreement](#accepting-the-contributor-license-agreement)
+- [Proposing changes to Ballerina](#proposing-changes-to-ballerina)
+
+## Getting engaged
+
+- The [Ballerina-Dev Google Group](#https://groups.google.com/forum/#!forum/ballerina-dev) is the main Ballerina project discussion forum. Also, you can use [StackOverflow](#https://stackoverflow.com/questions/tagged/ballerina) for support, [Slack](#https://app.slack.com/client/T47EAEKB5/DF818FX9Q) for real-time communications.
+
+- **IMPORTANT:** Send an email to security@ballerina.io to report sensitive security-related issues. For details, see the <a href="https://ballerina.io/security/">security policy</a>.
+
+- Submitting a bug is just as important as contributing to code. Click the **Issues** tab of the below GitHub repos, click **New Issue**, fill out all sections in the issue template, and submit the bug/issue report.
```

Comment:
The links are already for the issues tab, mentioning click the "Issues" tab is redundant.

Reply:
Changed to "Click **Issues** in the corresponding GitHub repo from the below ones,...".

Reply:
Still has the original issue I mentioned. For example, if you look at the first bullet point, the link is https://github.com/ballerina-platform/ballerina-lang/issues. 

When you click on that, you don't have to go and click "Issues" again. I suggest replacing
"Click the **Issues** tab of the below GitHub repos, click **New Issue**, fill out all sections in the issue template, and submit the bug/issue report."

with

"Create a new issue in the relevant GitHub repo:"

Reply:
Changed as follows:

"Create a new issue in the relevant GitHub repo using the below links, fill out all sections in the issue template, and submit the bug/issue report."